cli/cmd.go

```diff
@@ -81,6 +81,7 @@ var Commands = []*cli.Command{
 	WithCategory("developer", LogCmd),
 	WithCategory("developer", WaitApiCmd),
 	WithCategory("developer", FetchParamCmd),
+	WithCategory("developer", NonInteractiveAuthCmd),
```

Comment:
This definitely shouldn't be a root level-command.

Probably belongs in `walletCmd` if we don't want it in `lotus-shed`